cmd/challengepool.go

```diff
@@ -66,5 +66,5 @@ func init() {
 
 	cpInfo.PersistentFlags().String("allocation", "",
 		"allocation identifier, required")
-	cpInfo.Flags().Bool("json", false, "pass this option to print response as json data")
+	cpInfo.Flags().Bool("json (boolean)", false, "pass this option to print response as json data")
```

Comment:
It should be in the usage field, otherwise the flag would not work. we will see `unknown flag: --json`. Please consider to show the default value string by using the cobra template so that we can make the changes minimal.